### ISSUE TITLE
Accessibility issue in progress bar for pages that are not episodes: aria-valuenow="NA" - Invalid state or property

### ISSUE THREAD

**Issue by josenino95** [carpentries/varnish#201]
> An ARIA state or property has an invalid value. It may have been mistyped.

**Occurrences (1 in each page)**
From my understanding of the [SiteImprove report](https://my2.siteimprove.com/Inspector/1237935/A11Y/Page?pageId=343679300403&impmd=80DD58F77DAE7B7C0F08204A62F0ED91&issue=cantTell+failed&wcag=twopointone#/sia-r19/failed/RGY1eUJCNVQ=;ZDJiM2MyYWE=), the issue arises from the `aria-valuenow` property of the progress bar when it is set to the value of "NA".

I see this happening in the following pages:

- reference.html
- discuss.html
- CODE_OF_CONDUCT.html
- LICENSE.html
- resources.html
- any additional pages that are not episodes, for example this "Further Exercises" page in the SC Python lesson: https://swcarpentry.github.io/python-novice-gapminder/exercises.html

```
<div class="progress software">
<div class="progress-bar software" role="progressbar" aria-valuenow="NA" aria-label="Lesson Progress" aria-valuemin="0" aria-valuemax="100" style="width: NA%"></div>
</div>
```

> **Why is this an issue?**
> ARIA attributes provide information about the behavior of interactive elements. For example, they can be used to tell screen reader users whether a checkbox is checked (aria-checked=true) or unchecked (aria-checked=false).
> 
> Non-standard or misspelled values will not deliver the intended user experience and may even break the usability of the page.
> 
> WCAG 2.2: [Understanding Success Criterion 4.1.2: Name, Role, Value](https://www.w3.org/WAI/WCAG22/Understanding/name-role-value)
> ARIA 1.2: [6.2.4 Value](https://www.w3.org/TR/wai-aria-1.2/#propcharacteristic_value)
> Siteimprove Alfa: [Technical documentation for rule SIA-R19](https://alfa.siteimprove.com/rules/sia-r19)

**Comment by froggleston** [comment]
It looks like this is required for conformance (whilst #199 and #200 are best practice only) so I'll work on this next.
